### PR TITLE
[doc] Add known_issues file.

### PR DESCRIPTION
**Pull request by ejgallego**

### CODE REVIEW

**Comment by Zimmi48** [comment]
Thanks @ejgallego! If no one has any problem with there being such a file, we'll merge before releasing the beta. In the meantime, I suggest that anyone wanting to add known issues to this list push additional commits to [this branch](https://github.com/ejgallego/coq/tree/doc%2Bknown_issues).

File: KNOWN_ISSUES.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  backtracking such command may not undo its effects or create other
+  issues. We recommend users to group all the `Require`s at the
+  beginning of the document.
+- The `abstract` tactic may be fragile in certain situations, for
+  example inside a nested proof. We recommend using it only in
+  top-level proofs.
+- Interrupting Coq when using the XML protocol may produce ill-formed
+  XML. [BZ#5192](https://coq.inria.fr/bugs/show_bug.cgi?id=5192)
+- There are some known issues with the XML protocol, we recommend you
+  get in touch with the developers if you plan to use it.
```
Comment (Zimmi48) [inline KNOWN_ISSUES.md]: Would "get in touch with the developers via Coqdev or Gitter" be enough?

Comment (psteckler) [inline KNOWN_ISSUES.md]: I would give email/Web addresses for those, so someone doesn't have to find them.

File: KNOWN_ISSUES.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+Known Issues in Coq
+===================
+
+> Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.
+
```
Comment (psteckler) [inline KNOWN_ISSUES.md]: I believe that quote is attributable to de Saint-Exupery, not the Coq dev team. :-)

Comment (Zimmi48) [inline KNOWN_ISSUES.md]: Yes, let's give him credit.

**Comment by psteckler** [comment]
Should there be a reference to this document in the README, like there is for the new CONTRIBUTING document?

**Comment by Zimmi48** [comment]
Maybe I'm wrong but I view this more as a temporary file. A link from the 8.7beta announcement release will be sufficient, I guess.

**Comment by maximedenes** [comment]
We may want to go this road, but this list is too partial as is, and we should discuss it a bit I guess. So let's not integrate it in the beta, but it can still make it in 8.7.0 if necessary.

**Comment by Zimmi48** [comment]
My personal point of view on this was that it makes sense as a temporary thing during the beta but I wouldn't like this file to stay forever, because such files always end up not being updated. So I suggest we simply close the PR. Known issues which are going to stay after the 8.7.0 release and which are worth to be documented should be documented in the reference manual (or the developers' documentation when appropriate), as it is already the case with the async issue on Windows.

**Comment by ejgallego** [comment]
IMHO this should go in the release notes, not in the manual.

**Comment by maximedenes** [comment]
@Zimmi48 @ejgallego so what shall we do with this PR?

**Comment by Zimmi48** [comment]
I've told my opinion already:

> My personal point of view on this was that it makes sense as a temporary thing during the beta but I wouldn't like this file to stay forever, because such files always end up not being updated. So I suggest we simply close the PR. Known issues which are going to stay after the 8.7.0 release and which are worth to be documented should be documented in the reference manual (or the developers' documentation when appropriate), as it is already the case with the async issue on Windows.

Do what you want with it.

**Comment by ejgallego** [comment]
I am fine with closing this, however IMHO we need to agree on a method:

- to mark issues that are showstoppers for a release.
- to document known issues in releases [do release notes work here?]

**Comment by Zimmi48** [comment]
I agree. This would IMHO be worth considering in the larger setting of:
- what kind of releases notes do we want (e.g. what are our CHANGES, COMPATIBILITY, Credits policy...)
- what is the future of our issue tracker (I'm going to propose a move to GH issues at the next WG and this could be the occasion to review the way we give priority levels to issues, how we associate them with version milestones, etc.)

**Pull request closed without merging**